db/services/movie_session.py

```
@@ -0,0 +1,34 @@
+from db.models import MovieSession, Movie, CinemaHall
+import datetime
+
+def create_movie_session(movie_show_time, movie_id, cinema_hall_id):
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

In the `create_movie_session` function, using `Movie.objects.get(id=movie_id)` and `CinemaHall.objects.get(id=cinema_hall_id)` will raise a `DoesNotExist` exception if the movie or cinema hall is not found. Consider handling these exceptions to provide a more robust function.